dev/dags/dbt/altered_jaffle_shop/dbt_project.yml

```diff
@@ -0,0 +1,26 @@
+name: 'jaffle_shop'
+
+config-version: 2
+version: '0.1'
+
+profile: 'jaffle_shop'
```

Comment:
The profile `jaffle_shop` does not exist in profile.yml shall we adjust the name to match one of the profile name in profile.yml?

Reply:
Addressed in 1d23cc7a6a7dd6f3e2c20033b6a4f88ef296f76c